MatrixKit/Models/Account/MXKAccount.m

```diff
@@ -94,6 +98,18 @@ @implementation MXKAccount
 @synthesize userTintColor;
 @synthesize hideUserPresence;
 
++ (void)load
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        initialSyncSilentErrorsHTTPStatusCodes = @[
+            @(504),
+            @(524)
+        ];
+        //  TODO: How about 522 and 599?
```

Comment:
You are right we can add them to cover any kind of timeout errors or server setup.